core/src/test/java/org/elasticsearch/action/admin/cluster/health/ClusterHealthResponsesTests.java

```diff
@@ -199,26 +49,13 @@ public void testIsTimeout() throws IOException {
     }
 
     public void testClusterHealth() throws IOException {
-        ShardCounter counter = new ShardCounter();
```

Comment:
why not leave these and see that they are properly exposed/not influencing the ClusterHealthResponse object?